examples/cdn-preact/copy.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
A good idea to `chmod +x copy.sh` and check in that change, permissions should persist.

Reply:
Same deal for the other `.sh` files in the other examples.

Reply:
I'll address this in https://github.com/CMSgov/design-system/pull/2408